docs/sources/get-started/_index.md

```diff
@@ -0,0 +1,74 @@
+
+| Action                                                           | Considerations                                                                                                                                                                                                                    |
+| ---------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Decide what kind of documentation deliverables you need          | What do you want a user to accomplish?                                                                                                                                                                                            |
+| Know which release life cycle stage the product or feature is at | Review the [Release cycle for Grafana Labs](/docs/release-life-cycle/) <br /> Check in at the [Grafana Community Slack #docs channel](https://grafana.slack.com/archives/CNCRV74GP) to learn about docs for each of these stages. |
+| Consider content reuse, if applicable                            | Read the [Reuse strategy documentation]({{< relref "../write/reuse-content/reuse-directories" >}}) <br />Check in at #docs to find out how the docs team can support you in this.                                                 |
+
+### 2. Create the structure
+
+| Action                             | Considerations                                                         |
+| ---------------------------------- | ---------------------------------------------------------------------- |
+| Add docs to your GitHub repository | Add a `docs/sources` folder to your repo, if it doesn’t already exist. |
+| Create the folders and structure   | Use the content framework for consistency across product docs.         |
+
+### 3. Draft the docs
+
+| Action                                         | Considerations                                                                                                                                                                                   |
+| ---------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Learn how to best write and organize your docs | Read the [topic types]({{< relref "../structure/topic-types" >}}) topic and sub-topics in the Writers’ Toolkit.                                                                                  |
+| Add front matter                               | [Front matter]({{< relref "../write/front-matter" >}})der, and other key information to Hugo (our publishing platform). Learn more about [Front matter]({{< relref "../write/front-matter" >}}). |
```

Comment:
```suggestion
| Add front matter                               | [Front matter]({{< relref "../write/front-matter" >}}), and other key information to Hugo (our publishing platform). Learn more about [Front matter]({{< relref "../write/front-matter" >}}). |
```

Reply:
I'm making this change in the file directly (since I have it open to address some table-related issues).